### DISCUSSION TITLE
Interactive flowchart on Pluto?

### DISCUSSION THREAD

**Discussion by a9lara** [fonsp/Pluto.jl#853] [Questions about using Pluto] [answered]
Is it possible to implement an interactive flowchart on Pluto? The idea is that being on the flowchart you can select the specific part

Regards

Armando

**Comment by fonsp** [comment, marked as answer]
Complex inputs like this are definitely possible inside Pluto, take a look at the rectange example in the Interactivity sample notebook to learn how you can make custom inputs. You will need to use javascript.
